src/qibolab/backends.py

```diff
@@ -50,6 +51,33 @@ def __init__(self, platform):
         }
         self.compiler = Compiler.default()
 
+    @property
+    def qubits(self) -> list[Union[str, int]]:
```

Comment:
You can also use
```suggestion
    def qubits(self) -> list[QubitId]:
```
(needs `from qibolab.qubits import QubitId`).

Reply:
[020f980](https://github.com/qiboteam/qibolab/pull/1076/commits/020f98042c66909c5ad09ee82e7f0f749c35a8a3)